cli/package.json

```diff
@@ -1 +1,35 @@
-{"version": "6.7.0"}
\ No newline at end of file
+{
+  "name": "protobufjs-cli",
+  "description": "Translates between file formats and generates static code as well as TypeScript definitions.",
+  "version": "7.0.0",
```

Comment:
Just wanted to confirm that we want to start this separate package from v7.0.0 - it's up to @dcodeIO to decide.

Reply:
hey @dcodeIO can you advise here so that we can get this merged, if all else looks good. Thanks